index.html

```diff
@@ -4,49 +4,250 @@
+                <div class="tags2">
+                   <button class="tag-texts2">css</button>
+                </div>            
+                <div class="tags3">
+                   <button class="tag-texts3">javascript</button>
+                </div>            
+    
+            </div>
+            <div class="action2">
+                <div class="normal-button3">
+                    <button class="action-text3">see project</button>
+                </div>
+            </div>
+        </div>
+            
+        </div>
+    </div>
+    <div class="page4">
+        <div class="snapshoot3">
+        <img src="./image/Snapshoot Portfolio4.svg" alt="snap">
+        <div class="left-block2">
+            <div class="primary-text21"></div>
+        <h2 class="project-title2">Tonic</h2>
+        <ul class="frame2">
+            <div class="client2"><li class="text21">canopy</li></div>
+            <div class="role2"><li class="text22">Back End Dev</li></div>
+            <div class="year2"><li class="text23">2015</li></div>
         </ul>
+        <div class="primary-text22">
+        <p> A daily selection of privately personalised reads; no accounts or sign-up required. </p>
+        </div>
+        <div class="tags">
+            <div class="tags1"><button class="tag-texts1">html</button></div>
+            <div class="tags2">
+               <button class="tag-texts2">css</button>
+            </div>            
+            <div class="tags3">
+               <button class="tag-texts3">javascript</button>
+            </div>            
+
+        </div>
+        <div class="action2">
+            <div class="normal-button4">
+                <button class="action-text4">see project</button>
+            </div>
         </div>
     </div>
+       
     </div>
-  </section>
+</div>
+<div class="about-myself">
+    <div class="top">
+        <h2 class="text-intro">About myself</h2>
+        <p class="text-description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of 
+            my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <div class="social-media">
+                <div class="supporting-text">
+                    <h4>LET'S CONNECT</h4>
+                </div>
+                <ul class="network">
+                    <li><a href="https://twitter.com/AimeSebagabo"><i class='bx bxl-twitter'></i></a></li>
 
+                    <li><a href="https://www.linkedin.com/in/jean-aime-sebagabo-279aa2207/"></a><i class='bx bxl-linkedin'></i></a></li>
+
+                    <li><a href=""><i class='bx bxl-gmail'></i></a></li>
+
+                    <li><a href="https://github.com/Hazard10-eng"><i class='bx bxl-github'></i></a></li>
+                </ul>
+                <div class="resume">
+                   <input type="text" value="Get my resume" class="text-resume">
+                </div>
```

Comment:
Kindly make the button "Get my resume" a link `a` or `button` tag, and let's also add the button styling: `Active`, `Hover` and `Pressed`.